cni/network/network.go

```diff
@@ -533,12 +530,12 @@ func (plugin *NetPlugin) Add(args *cniSkel.CmdArgs) error {
 			if err != nil {
 				return fmt.Errorf("IPAM Invoker Add failed with error: %w", err)
 			}
-			sendEvent(plugin, fmt.Sprintf("Allocated IPAddress from ipam:%+v v6:%+v", ipamAddResult.ipv4Result, ipamAddResult.ipv6Result))
+			sendEvent(plugin, fmt.Sprintf("Allocated IPAddress from ipam:%+v", ipamAddResult.defaultCniResult))
```

Comment:
You need to print the whole ipamAddResult (includeing the secondary interfaces here)